scrapy_po/webpage.py

```diff
@@ -7,17 +7,18 @@
 from .page_inputs import ResponseData
 
 
-class PageObject(abc.ABC):
+class Injectable(abc.ABC):
     """
-    All web page objects should inherit from this class.
-    It is a marker for a middleware to pick anargument and populate it.
+    Injectable objects are automatically built and passed as arguments to
+    callbacks that requires them. The ``InjectionMiddleware`` take care of it.
 
-    Instead of inheriting you can also use ``PageObject.register(MyWebPage)``.
+    Instead of inheriting you can also use ``Injectable.register(MyWebPage)``.
+    ``Injectable.register`` can also be used as a decorator.
     """
     pass
 
 
-class ItemPage(PageObject):
```

Comment:
PageObject disappears as an entity in the code. If we use the term in the comments, etc, we should make clear what "Page Object" means.